islandora_video_streaming_kaltura.module

```diff
@@ -81,4 +81,56 @@ function islandora_video_streaming_kaltura_process_islandora_video($variables =
   }
 }
 
+/*
+ * Implements hook_islandora_datastream_alter().
+ */
+function islandora_video_streaming_kaltura_islandora_object_alter(AbstractObject $object, &$context) {
+  if (isset($object['KALTURA_STREAM']) && ($context['action'] == 'purge' || $context['action'] == 'delete' )) {
+    $object_content = $object['KALTURA_STREAM']->content;
+    $decoded = json_decode($object_content, true);
+    if ($decoded !== null) {
+      $entity = kaltura_entry_load($decoded['entry_id']);
```

Comment:
You are assuming `$decoded['entry_id']` is set.  What happens if it is not?

Reply:
An error or exception occurs in the later code statements. Would it be better to have 
`if ($decoded !== null && isset($decoded['entry_id']) )`?